cpp/examples/RawConnection.cpp

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
```

Comment:
The users couldn't use the connection directly.  the example is for session and pool

Reply:
I will provide the example for session later.

Reply:
so you can delete it

Reply:
Ok, fixed.